examples/vic-gov-au/test/e2e/plugins/index.js

```diff
@@ -1,15 +1,15 @@
 const cucumber = require('cypress-cucumber-preprocessor').default
-const tideAdmin = require('@dpc-sdp/ripple-test-tools')
 
 // Environment variables that need exposing to cypress go here - use the example site .env file
 require('dotenv').config()
 
+let momentNow
+
 module.exports = (on, config) => {
   on('file:preprocessor', cucumber())
 
-  tideAdmin(on, config)
-
   config.env = {
+    SITE_ID: '4',
```

Comment:
Do we need this in each project? If so we need to make it configurable, as most of the project site id is not `4`.

Reply:
added automatically with create-ripple-app